packages/protocol/contracts/common/GoldToken.sol

```diff
@@ -135,6 +135,23 @@ contract GoldToken is Initializable, CalledByVm, Freezable, IERC20, ICeloToken {
     return true;
   }
 
+  /**
+   * @notice Mints new cGLD and gives it to 'to'.
+   * @param to The account for which to mint tokens.
+   * @param value The amount of cGLD to mint.
+   */
+  function mint(address to, uint256 value) external onlyVm onlyWhenNotFrozen returns (bool) {
```

Comment:
Should this be `onlyWhenNotFrozen`? Otherwise epoch rewards will be frozen until gold transfers are unfrozen

Reply:
I though they were the same thing.. I'll remove that